src/main/kotlin/dev/binclub/paperbin/transformers/TpsCompensation.kt

```diff
@@ -110,7 +110,7 @@ object TpsCompensation: PaperBinFeature {
 					}
 					//PUTFIELD .e : I
 				}
-			}
+			}*/

```

Comment:
Why are these changes relevant to this PR? Please make a new PR for these changes

Reply:
oops